src/app/common/utils/sort-assets-by-symbol.ts

```diff
@@ -0,0 +1,23 @@
+export interface Asset {
+  name: string;
+  icon: string;
+}
+
+export function sortAssetsBySymbol(assets: Asset[]) {
```

Comment:
This is basically the same as `sortSwappableAssetsBySymbol` with a different type. 

I need to refactor it but I was feeling the heat of getting this done quickly and getting frustrated with typescript.

Reply:
Ok, cool, I am actually in the middle of refactoring our use of the alex-sdk queries so I can tackle this in my refactor after I rebase on it.

Reply:
I have already relocated this on my branch, so will be easy to address there.

Reply:
That's great, thanks! I am not the best with TS so things like this can take me a while to figure out. I tried having a union type and extending types but just wanted to get it done quick so loosened it